File: CS50x/PSet6/sentimental-mario-more/mario.py
from cs50 import get_int

# ask height
height = get_int("Height: ")
while height <= 0 or height >= 9:
    height = get_int("Height: ")
# number os spaces and blocks in first line
space = height - 1
block = 1

# for each line, print spaces and blocks, adding a blocks and decreasing a space
for i in range(height):
    spaces = space * " "
    blocks = block * "#"
    print(spaces + blocks + "  " + blocks)
    space -= 1
    block += 1

File: CS50x/PSet1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
    //declaring variable
    int height;
    do
    {
        // asking height...
        height = get_int("Height: ");
    }
    // ...between 1-8
    while (height < 1 || height > 8);

    // for each line...
    for (int i = 0; i < height; i++)
    {
        // ...print spaces
        for (int j = height - i; j > 1; j--)
        {
            printf(" ");
        }
        // ...print hashs
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}
